src/pages/resultsView/download/DownloadUtils.spec.ts

```diff
@@ -664,4 +664,252 @@ describe('DownloadUtils', () => {
                 "protein data is correct for the sample key VENHQS1FRS1BMjBDLTA2OnNrY21fdGNnYQ");
         });
     });
+
+    describe('updateOqlData', () => {
+        it('properly updates oql data', () => {
+            const sampleOql = {
```

Comment:
lets call this a sampleOqlReport.   Try to include the type/identify of the entity in its name.  For example, this variable does not contain oql, it contains a report about oql.